### PR TITLE
Feature/excercises tijmen

### PR DESCRIPTION
**Pull request by tuvaa**

### CODE REVIEW

**Review by basvdbrink — APPROVED** [review]

File: lab1/tijmen_12610305/Excercise7.hs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+-- Via card check, rules: https://www.creditcard.nl/faq/creditcardnummer
+isVisa :: Integer -> Bool
+isVisa n = luhn n
+        && (length xs == 13 || length xs == 16)
+        && head xs == 4
+        where xs = intToList n
+
+excercise7 :: IO()
+excercise7 = do
+  putStrLn "Work in progress"
```
Comment (basvdbrink) [inline lab1/tijmen_12610305/Excercise7.hs]: I'm assuming we will figure this out when merging our work

File: lab1/tijmen_12610305/Excercise7.hs [hunk 2]
```diff
@@ -0,0 +1,52 @@
+-- Time spent: 1u30m
+module Excercise7 where
```
Comment (basvdbrink) [inline lab1/tijmen_12610305/Excercise7.hs]: Isn't this a typo? Shouldn't it be `exercise`?

Comment (tuvaa) [reply lab1/tijmen_12610305/Excercise7.hs]: Heh, it is :D

File: lab1/tijmen_12610305/Excercise7.hs (hunk 1 above)
Comment (tuvaa) [reply lab1/tijmen_12610305/Excercise7.hs]: It's no longer a WIP - I added the tests in a later commit.

**Pull request merged**